docusaurus/docs/reference/tunnelers/linux/linux-tunnel-options.md

```diff
@@ -69,3 +69,78 @@ If the DNS record exists it returns the answer and sets query status to `NO_ERRO
 #### System Requirements for Mode `run-host`
 
 `ziti-edge-tunnel run-host` does not require elevated privileges or the above device or socket, only network egress to the servers for which it is hosting Services.
+
+## Multi-Factor Authentication
+
+You may use multi-factor authentication (MFA) with `ziti-edge-tunnel` by completing the following workflow to register a time-based one-time password (TOTP) token generator with the OpenZiti Controller.
+
+### Find the Identifier of the OpenZiti Identity
+
+The identifier is typically the full path to the JSON identity file that is already added to the running `ziti-edge-tunnel run` process.
```

Comment:
Same comment. Inform the reader what it it not what it typically is and then inform them how do get the value